xbmc/filesystem/MusicDatabaseDirectory.cpp

```diff
@@ -166,9 +166,10 @@ void CMusicDatabaseDirectory::ClearDirectoryCache(const std::string& strDirector
 
 bool CMusicDatabaseDirectory::IsAllItem(const std::string& strDirectory)
 {
-  //Last query parameter, ignoring any appended options, is -1
+  //Last query parameter, ignoring any appended options, is -1 or -2
   CURL url(strDirectory);
-  if (StringUtils::EndsWith(url.GetWithoutOptions(), "/-1/"))
+  if (StringUtils::EndsWith(url.GetWithoutOptions(), "/-1/") || // any albumid
```

Comment:
Maybe move this to constexpr string_views to avoid the magic numbers?

Reply:
The video/music code is full of magic numbers. I would like to postpone this larger cleanup to another day/PR.